lib/ClimaCoreTempestRemap/src/netcdf.jl

```diff
@@ -228,10 +228,19 @@ function def_space_coord(
     type = "dgll",
 ) where {S <: Spaces.Staggering}
     hvar = def_space_coord(nc, space.horizontal_space; type = type)
-    vvar = def_space_coord(
-        nc,
-        Spaces.FiniteDifferenceSpace{S}(space.vertical_topology),
-    )
+    if space.vertical_topology isa Topologies.IntervalTopology
```

Comment:
vertical_topology = space.vertical_topology isa
           ClimaCore.Hypsography.TerrainWarpedIntervalTopology ? space.vertical_topology.topology : space.vertical_topology
    vvar = def_space_coord(
        nc,
        Spaces.FiniteDifferenceSpace{S}(vertical_topology),
    )

Reply:
In general, can we just use `space.vertical_topology`. Switching between `space.vertical_topology` and `space.vertical_topology.topology` does not seem to be intuitive